admin/src/js/controllers/images-branding.js

```diff
@@ -3,111 +3,127 @@ angular.module('controllers').controller('ImagesBrandingCtrl',
     $log,
     $scope,
     $translate,
-    DB
+    AddAttachment,
+    DB,
+    Translate
   ) {
 
     'ngInject';
     'use strict';
 
-    const BRANDING_ID = 'branding';
+    const DOC_ID = 'branding';
+    const MAX_FILE_SIZE = 100000; // 100MB
```

Comment:
Is this really 100MB? I'm seeing the warning appear for a 5MB file.

Reply:
You're right. It's 100KB. Updated the comment and error message below.